### PR TITLE
fixed league nav bar

### PR DESCRIPTION
**Pull request by jjaykim**
## desktop version
Screen Shot 2021-11-12 at 6 19 47 AM

## mobile version
Screen Shot 2021-11-12 at 6 19 41 AM

### CODE REVIEW

File: client/src/components/navigation/components/LeagueNav.tsx [hunk 1]
```diff
@@ -24,6 +24,18 @@ export const LeagueMobileNav: React.FC<LeagueMobileNavProps> = ({
...
+  const leagueName = useMemo(() => {
+    if (!viewer.leagues) return "Comming Soon";
+    return (
+      viewer.leagues[0].year +
+      ' ' +
+      viewer.leagues[0].name +
+      ' ' +
+      viewer.leagues[0].leagueType
+    );
+  }, [viewer.leagues])
```
Comment (koreancanadiansoccer) [inline client/src/components/navigation/components/LeagueNav.tsx:37]: use string literal;
`${viewer.leagues[0].year} ${viewer.leagues[0].name} ${viewer.leagues[0].leagueType}`

File: client/src/components/navigation/components/LeagueNav.tsx [hunk 2]
```diff
@@ -76,6 +82,17 @@ export const LeagueNav: React.FC = () => {
...
+
```
Comment (koreancanadiansoccer) [inline client/src/components/navigation/components/LeagueNav.tsx:95]: same here

File: client/src/components/navigation/components/LeagueNav.tsx (hunk 1 above)
Comment (jjaykim) [reply client/src/components/navigation/components/LeagueNav.tsx:31]: fixed!

File: client/src/components/navigation/components/LeagueNav.tsx (hunk 2 above)
Comment (jjaykim) [reply client/src/components/navigation/components/LeagueNav.tsx:83]: fixed!

**Review by dhmoon91 — APPROVED** [review]

**Pull request merged**